src/context/ThemeContext.tsx

```diff
@@ -0,0 +1,36 @@
+import { createContext, useState, useContext, useEffect } from 'react';
+
+type Theme = {
+	theme: 'light' | 'dark';
+	setTheme: React.Dispatch<React.SetStateAction<'light' | 'dark'>>;
+};
+
+const ThemeContext = createContext<Theme>({
+	theme: 'light',
+	setTheme: () => null,
+});
+
+// const ThemeContext = createContext({});
+
+export const ThemeProvider: React.FC = ({ children }) => {
+	const [theme, setTheme] = useState<'light' | 'dark'>('light');
```

Comment:
You could persist the selected value with the help of local storage.

Reply:
Yes that's in the plan, will do when doing backend and storing the userData